src/style.css

```
@@ -0,0 +1,112 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url("../src/fonts/Avenir-Book.ttf") format("ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-H";
+  src: url("../src/fonts/Avenir-Heavy.ttf") format("ttf");
+  font-weight: 300;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
```

Only box-sizing `border-box` allows for `*` selector
```suggestion
* {
  padding: 0;
  box-sizing: border-box;
}
```